_rules/device-motion-disabled-c249d5.md

```diff
@@ -0,0 +1,205 @@
+---
+id: c249d5
+name: Device motion based functionality can be disabled
```

Comment:
This isn't testing that functionality can be disabled; you're not checking that the event handler is used to operate "functionalty".

Reply:
Updated name and description